### DISCUSSION TITLE
doco says set table column width

### DISCUSSION THREAD

**Discussion by dubbya** [Budibase/budibase#4917] [Help] [answered]
https://docs.budibase.com/docs/table says "Change the width of each column" 

I'm probably missing something obvious but how this is done?

**Comment by mjashanks** [comment, marked as answer]
Hi @dubbya.

You can change the width of a column by 
1. Selecting your table
2. Clicking on "Configure Columns"
3. Clicking the settings button, beside your column. You may need to "Add All columns" before you can see each individual column.

![image](https://user-images.githubusercontent.com/3524181/158560186-500f75c0-fb8c-461f-85f6-d82d433e9ecc.png)

We have only just released this feature, so you might find that you don't see it in your app. If so, you probably need to update your app. You can do so with the button in the top-right of the platform:

![image](https://user-images.githubusercontent.com/3524181/158560562-2f4b70ba-d65e-4af1-a032-991395d65a2b.png)

**Reply by dubbya** [reply]
Shanks very much @mjashanks ;) the update was it. 

I had to do a CLI budi hosting --update and then click that button at the top-right to get the update otherwise it just told me I was on the current version 1.0.73.
